lnrpc/chainrpc/chainnotifier.proto

```diff
@@ -14,7 +16,12 @@ service ChainNotifier {
     particular transaction by its hash or for an output script by specifying a
     zero hash.
     */
-    rpc RegisterConfirmationsNtfn (ConfRequest) returns (stream ConfEvent);
+    rpc RegisterConfirmationsNtfn (ConfRequest) returns (stream ConfEvent) {
+        option (google.api.http) = {
+            post: "/v2/chainnotifier/register_confirmations_ntfn"
```

Comment:
Could simplify these paths to: `register/confirmation`, `register/spend`, `register/blocks`.